src/types.ts

```diff
@@ -3,14 +3,20 @@ import type { ec as EC } from 'elliptic';
 import type { BigNumberish } from './utils/number';
 
 export type KeyPair = EC.KeyPair;
-export type Signature = EC.Signature;
+export type Signature = BigNumberish[];
 
 export type GetContractAddressesResponse = {
   Starknet: string;
   GpsStatementVerifier: string;
 };
 
-export type Status = 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'REJECTED' | 'ACCEPTED_ONCHAIN';
+export type Status =
+  | 'NOT_RECEIVED'
+  | 'RECEIVED'
+  | 'PENDING'
+  | 'ACCEPTED_ON_L1'
```

Comment:
Does not matter but `ACCEPTED_ON_L1` should come after `ACCEPTED_ON_L2` for consistency.